
export const getOne = (req, res) => {
    try {
        
    } catch (error) {
        
    }
}